src/Matterhorn/State/Messages.hs

```diff
@@ -681,6 +686,57 @@ runNotifyCommand post mentioned = do
                                  [notified, sender, messageString] Nothing Nothing
                 return Nothing
 
+encodeToJSONstring :: A.ToJSON a => a -> String
+encodeToJSONstring a = map (chr . fromEnum) . BL.unpack $ A.encode a
```

Comment:
Rather than `map (...) . BL.unpack`, please use [Data.ByteString.Char8.unpack](https://hackage.haskell.org/package/bytestring-0.11.1.0/docs/Data-ByteString-Char8.html#v:unpack) to go directly to `String`.

Reply:
Resolved by commit f96534a4 .

Reply:
Resolved by commits f96534a and adc90b78 .